integration-tests/src/tests/client/features/nearvm.rs

```diff
@@ -90,6 +92,6 @@ fn test_wasmer2_upgrade() {
         capture.drain()
     };
 
-    assert!(logs_at_old_version.iter().any(|l| l.contains(&"vm_kind=Wasmer0")));
-    assert!(logs_at_new_version.iter().any(|l| l.contains(&"vm_kind=Wasmer2")));
+    assert!(logs_at_old_version.iter().any(|l| l.contains(&"vm_kind=Wasmer2")));
+    assert!(dbg!(logs_at_new_version).iter().any(|l| l.contains(&"vm_kind=NearVm")));
```

Comment:
I don't think it's a problem to print some extra stuff to stderr in tests but is this `dbg!` intended? It looks a bit odd to have it for `logs_at_new_version` but not for `logs_at_old_version`

Reply:
Yeah, I intentionally try to keep any debug statements that are necessary to diagnose a test failure I encounter, given that it may fail the same way in the future. In this particular case the wasmer2 test didn't stay valid due to https://github.com/near/nearcore/issues/8590, and I ended up discovering this by looking at just the new logs. The old protocol version on the other hand behaved predictably (to me,) so I didn't need the logs to verify it works the way it should.

Reply:
Ok, makes sense. Probably, for consistency, I would have added it to the old protocol version, too. (For someone else, it might also fail there?) But since it's already merged I would keep it as it is.